book/04-git-server/sections/protocols.asc

```diff
@@ -3,15 +3,21 @@
 Git can use four major protocols to transfer data: Local, HTTP, Secure Shell (SSH) and Git.
 Here we'll discuss what they are and in what basic circumstances you would want (or not want) to use them.
 
+[NOTE]
+====
+In addition to the above, both FTP and FTPS protocols are supported, but those protocols
+are inefficient and deprecated, and won't be discussed here.
+====
+
```

Comment:
I don't think we need this note here. By not talking about FTP, we effectively communicate that it's not a "major protocol," and that using it shouldn't be relied upon.

Reply:
Fair enough, I just mentioned it since it's in the man page for "git clone" and readers might wonder why it's not mentioned in the book. Will remove.